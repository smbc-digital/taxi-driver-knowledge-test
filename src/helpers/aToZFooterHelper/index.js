const showAToZFooter = (show) =>{
    let footer  = document.getElementById('atoz')
    if(footer !== null) 
    {
        if(show){
            footer.classList.remove('hidden')
        }
        else{
            footer.classList.add('hidden')
        }
    }
}

export default showAToZFooter