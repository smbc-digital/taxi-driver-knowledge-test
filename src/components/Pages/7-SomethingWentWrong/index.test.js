import { React } from '../../../helpers/SetupTest'
import { SomethingWentWrong } from './index'
import renderer from 'react-test-renderer'

describe('SomethingWentWrong', () => {
    describe('snapshot', () => {
		it('should render correctly',() => { 
			// Arrange
			const history = { block: jest.fn() }
			const location = {search: {

			}}

			// Act
			const tree = renderer
			.create(<SomethingWentWrong history={history} location={location} />)
			.toJSON()
		
			// Assert
			expect(tree).toMatchSnapshot()
		})
	})
})
