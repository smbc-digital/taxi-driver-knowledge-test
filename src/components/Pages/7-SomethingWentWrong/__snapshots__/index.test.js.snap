// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomethingWentWrong snapshot should render correctly 1`] = `
<form>
  <div
    className="ineligible-warning"
  >
    <div
      className="header-container"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-exclamation"
        />
      </div>
      <h1>
         We're sorry, there was a problem with the form
      </h1>
    </div>
    <p
      className="no-margin-bottom"
    >
      No money has been taken from your account, please try again if you want to continue.
    </p>
    <a
      className="button-primary"
      href="test"
    >
      Go back to try the payment again
    </a>
    <a
      className="button-secondary anchor-as-block"
      href="https://www.stockport.gov.uk/"
    >
      Go to the homepage
    </a>
  </div>
</form>
`;
